docs/docs/30-how-to-guides/20-managing-credentials.md

```diff
@@ -378,27 +378,54 @@ to the required ECR repositories.
 
 :::caution
 This method of authentication is a "lowest common denominator" approach that
-will work regardless of where Kargo is deployed. i.e. If running Kargo outside
-of EKS, this method will still work.
+will work regardless of where Kargo is deployed. i.e. if running Kargo outside EKS, this method will still work.
 
-If running Kargo within EKS, you may wish to consider using EKS Pod Identity
+If running Kargo within EKS, you may wish to either consider using EKS Pod Identity or IRSA
 instead.
 :::
 
-#### EKS Pod Identity
+#### EKS Pod Identity or IAM Roles for Service Accounts (IRSA)
 
 If Kargo locates no `Secret` resources matching a repository URL, and if Kargo
-is deployed within an EKS cluster, it will attempt to use
+is deployed within an EKS cluster, it will first attempt to use
 [EKS Pod Identity](https://docs.aws.amazon.com/eks/latest/userguide/pod-identities.html)
 to authenticate, but this relies upon some external setup. Leveraging this
 option eliminates the need to store credentials in a `Secret` resource.
 
-First, follow
+If you opt to use EKS Pod Identity, follow
 [this overview](https://docs.aws.amazon.com/eks/latest/userguide/pod-identities.html#pod-id-setup-overview)
 to set up EKS Pod Identity in your EKS cluster and assign an IAM role to the
 `kargo-controller` `ServiceAccount` within the `Namespace` to which Kargo is (or
 will be) installed.
 
+If you opt to instead use IRSA, follow [this overview](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) 
+to set up IRSA in your EKS cluster and assign an IAM role to the
+`kargo-controller` `ServiceAccount` within the `Namespace` to which Kargo is (or
+will be) installed.
```

Comment:
Suggested edits:

```
If Kargo locates no `Secret` resources matching a repository URL and  is deployed
within an EKS cluster, it will attempt to use
[EKS Pod Identity](https://docs.aws.amazon.com/eks/latest/userguide/pod-identities.html)
or
[IAM Roles for Service Accounts (IRSA)](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html)
to authenticate. Both of these rely upon some external setup. Leveraging either
eliminates the need to store ECR credentials in a `Secret` resource.

Follow
[this overview](https://docs.aws.amazon.com/eks/latest/userguide/pod-identities.html#pod-id-setup-overview)
to set up EKS Pod Identity in your EKS cluster or
[this one](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to set up IRSA.
For either, you will assign an IAM role to the `kargo-controller` `ServiceAccount`
within the `Namespace` to which Kargo is (or will be) installed.
```

Reply:
The main thing here is I don't want it to flow like, "First we try this and _then_ we try that," since that isn't the case. If either is set up, it will just work.